nanopub/nanopub.py

```diff
@@ -8,84 +8,198 @@
 import requests
 from rdflib.namespace import RDF, DC, DCTERMS, XSD
 
-from nanopub import java_wrapper
+from nanopub import java_wrapper, namespaces
+
+DEFAULT_URI = 'http://purl.org/nanopub/temp/mynanopub'
 
 
 class Nanopub:
```

Comment:
Now that I added the `from_assertion` method, this `Nanopub` class is starting to feel a lot more like a wrapper for RDF... That rings a bell right? We could subclass the `RdfWrapper` class from fairworkflows, or even move the Nanopub class there (although that seems kind of strange). Preferably we should do that in a separate issue. What do you think?

Reply:
This is a good point. Although given that the nanopub lib will be used in a number of non-fairworkflows related places as well (hopefully) then maybe it's still better that it just return the raw ```rdflib``` graph, rather than wrapping it all up. However, I am also wondering now... are there any other libraries out there that provide a powerful class that looks like ```RdfWrapper``` that we could use for both libs?

Reply:
OK, we'll just stick with this for now. Good point about existing libraries having some sort of `RdfWrapper`. I did a quick search (for example [https://wiki.python.org/moin/RdfLibraries](here)), but couldn't really find anything.